templates/iot/examples/infinispan/manual/050-StatefulSet.yaml

```diff
@@ -91,7 +91,7 @@ spec:
         volumeMounts:
         - mountPath: /opt/jboss/infinispan-server/standalone/configuration/custom
           name: infinispan-configuration
-        - mountPath: /opt/jboss/infinispan-server/standalone/data
+        - mountPath: /opt/infinispan/server/data/
```

Comment:
this looks like infinispan 10 image path

Reply:
Yes, but I changed to config as well, that should work.